web/src/app/users/UserContactMethodEditDialog.tsx

```diff
@@ -96,3 +102,15 @@ export default function UserContactMethodEditDialog({
     />
   )
 }
+
+export default function UserContactMethodEditDialogSwitch(
```

Comment:
not sure what's happening but the whole page flickers when i open the edit dialog from profile. seeing in stage too so not blocking but would be nice to have fixed

Reply:
Looks like this issue is on prod as well. created an issue for it since it is unrelated to this PR https://github.com/target/goalert/issues/3751